ts/components/leftpane/LeftPaneSettingSection.tsx

```diff
@@ -42,42 +42,42 @@ const StyledSettingsListItem = styled.div<{ active: boolean }>`
   }
 `;
 
-const getCategories = () => {
+const getCategories = (): Array<{ id: SessionSettingCategory; title: string }> => {
   return [
     {
-      id: SessionSettingCategory.Privacy,
+      id: 'privacy' as const,
```

Comment:
Is this because of the enum issue?

Reply:
as const because otherwise that 'privacy' string is resolved as a string, which we don't want. We want a `SessionSettingCategory` which cannot be all type of strings,
```ts
const getCategories = (): Array<{ id: SessionSettingCategory; title: string }> => {
  return [
    {
      id: 'privacy' as const,
      ```

Reply:
Okay cool. I'm surprised VS Code isn't smart enough to pick that up from the return type but all good. Resolved